nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "watch": ["src"],
+    "ext": "ts",
+    "ignore": ["src/public"],
+    "exec": "NODE_ENV=development npm start"
```

Comment:
Doing `npm start` on every ts change is not optimal. It might make sense to extract TS>JS transpiration to separate task that is run upon change of TS file. Alternatively `ts-node` could be used directly.

Since you engaged `gulp` I suggest you extract new TS task that will be used on TS source change.

Reply:
i'm removing nodemon for the time being as this can be done with gulp